conf/application.conf

```diff
@@ -17,6 +17,11 @@ github {
   clientSecret=${?GITHUB_APP_CLIENT_SECRET}
 }
 
+travis.token {
+  opensource=${?PROUT_TRAVIS_TOKEN_OPENSOURCE}
+  commercial=${?PROUT_TRAVIS_TOKEN_COMMERCIAL}
```

Comment:
@jacobwinch we'll need to generate a Travis CI token for `travis-ci.com` and set it as a  `PROUT_TRAVIS_TOKEN_COMMERCIAL` environment variable for the Prout instance.